Dockerfile.dev

```diff
@@ -0,0 +1,63 @@
+ARG NODE_VERSION
+
```

Comment:
This Dockerfile looks good, but here are some of my opinions:
- Node version should be pinned.
- This can be simplified by basing it on the Next.js example Dockerfile. You can find an example [here](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile).
- Use `standalone output` for reduced  image size. Use a boolean environment variable like `DOCKER_OUTPUT` to enable this conditionally. You can find more information [here](https://nextjs.org/docs/app/api-reference/next-config-js/output#automatically-copying-traced-files).

Reply:
Thanks...have to fix this.